client/src/components/Intro/index.tsx

```diff
@@ -92,7 +92,7 @@ const Intro = ({
               sameTab={false}
               to='/donate'
             >
-              {t('buttons.support-our-nonprofit')}
+              {t('buttons.donate')}
```

Comment:
Although more people click on 'support-our-nonprofit', more people who click on 'buttons.donate' donate. 🤷

Reply:
Can we remove the key from the i18n object? It looks like this is the only place it is used.

Reply:
yes that is what I was going to ask. I will remove it and push.

Reply:
FYI: there's almost no cost to removing unused keys.  It cleans up the codebase (which I'm always keen on) and the translations will stick around in the memory.